Assignment2/MMN44-fromRDFtoJSON-LD.jsonld

```diff
@@ -0,0 +1,48 @@
+{
+  "@context": {
+    "vocab": "http:www.example.org/ontology#",
+    "Class01": "vocab:class",
+    "Sensor029": "vocab:sensor",
+    "Measurement8401": "vocab:measurement",
+    "Computer101": "vocab:computer",
+    "User10A": "vocab:user",
+    "prop": "http://www.example.org/properties#",
+    "includes":"prop:includes",
+    "hasMeasurement": "prop:hasMeasurement",
+    "hasTemperature": {
+      "@id": "prop:hasTemperature",
+      "@type": "xsd:integer"
+    },
+    "atTime": {
+      "@id": "prop:atTime",
+      "@type": "xsd:dateTimeStamp"
+    },
+    "hasOwner": "prop:hasOwner",
+    "hasName": {
+      "@id": "prop:hasName",
+      "@type": "xsd:string"
+    },
+    "xsd": "http://www.w3.org/2001/XMLSchema#"
+  },
+  
+  "Class01": [{
+    "includes": "Sensor029",
+    "Sensor029": {
```

Comment:
You don't need to define "Sensor029". It isn't an attribute, it is a resource, thus I suggest you to remove it. Remember that when you define a resource, you only have to use curly braces and define inside all the resource attributes. However, when you have to specify a relationship between resources is when you have to define it as an attribute, for example as you did with the attribute "hasMeasurement" for defining the relationship between Sensor029 and Measurment8401. Note that "hasMeasurement" must be defined in the @context as you have done.
This comment also applies for "Measurement8401", "Computer101", and "User10A".